internal/services/mssql/mssql_server_security_alert_policy_resource_test.go

```diff
@@ -61,15 +68,32 @@ func (MsSqlServerSecurityAlertPolicyResource) Exists(ctx context.Context, client
 		return nil, err
 	}
 
-	resp, err := client.MSSQL.LegacyServerSecurityAlertPoliciesClient.Get(ctx, id.ResourceGroup, id.ServerName)
+	serverId := commonids.NewSqlServerID(id.SubscriptionId, id.ResourceGroup, id.ServerName)
+
+	result, err := client.MSSQL.ServerSecurityAlertPoliciesClient.Get(ctx, serverId)
 	if err != nil {
-		if utils.ResponseWasNotFound(resp.Response) {
+		if response.WasNotFound(result.HttpResponse) {
 			return nil, fmt.Errorf("SQL Security Alert Policy for server %q (Resource Group %q) does not exist", id.ServerName, id.ResourceGroup)
 		}
 		return nil, fmt.Errorf("reading SQL Security Alert Policy for server %q (Resource Group %q): %v", id.ServerName, id.ResourceGroup, err)
 	}
 
-	return utils.Bool(resp.ID != nil), nil
+	model := result.Model
+	if model == nil {
+		return nil, fmt.Errorf("reading SQL Security Alert Policy for server %q (Resource Group %q): Model was nil", id.ServerName, id.ResourceGroup)
+	}
+
+	return pointer.To(model.Id != nil), nil
+}
+
+// Sets the tests 'Primary' and 'Secondary' locations based on the 'ARM_MICROSOFT_TEST' environment variable due to subscription quota policies
+func setTestDataLocationBySubscription(data acceptance.TestData) acceptance.TestData {
```

Comment:
Similar to a previous comment from another PR, we can remove this and just update the teamcity settings

Reply:
Fixed.